classes/install/Installer.php

```diff
@@ -737,7 +737,7 @@ public function installEmailTemplate($installer, $attr)
     {
         $locales = explode(',', $attr['locales'] ?? '');
         // FIXME pkp/pkp-lib#6284 Remove after drop of support for upgrades from 3.2.0
-        if (!Schema::hasColumn('email_templates_default', 'stage_id')) {
+        if (Schema::hasTable('email_templates_default') && !Schema::hasColumn('email_templates_default', 'stage_id')) {
```

Comment:
I recall that PR. This Installer's method is triggered in upgrades of older versions and must remain here despite the table removal. Otherwise it will brake the migration, with a fatal error described in the issue: https://github.com/pkp/pkp-lib/issues/6284#issue-722189268
This is why we are avoiding using DAO calls during migration, like in this method below.

Reply:
On second thought, you are right. I've already modified the email template DAO.